.github/workflows/release.yml

```diff
@@ -302,6 +304,7 @@ jobs:
           git fetch
           git checkout ${{ github.ref_name }}
           git push --force origin refs/heads/${{ github.ref_name }}:refs/heads/latest
+          git push --force origin refs/heads/${{ github.ref_name }}:refs/heads/docs-latest
```

Comment:
The one thing that makes me nervous about this is the possibility of losing doc-fixes that somehow got cherry-picked to `docs-latest` but didn't get merged back into main. I think the convenience is probably a fair trade-off, but something to make sure everybody is aware of and why it is important that we only cherry-pick to this branch and never commit to it directly.

Reply:
I hope the warning in `docs/README.md` is enough to prevent this from happening. Will also post a PSA in Slack